specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-12-01/Marketplace.json

```diff
@@ -0,0 +1,897 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-12-01",
+    "title": "MarketplaceManagementClient",
+    "description": "REST APIs for Azure Marketplace"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/notifications": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_Notifications",
+        "description": "Get all private store notifications",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreNotifications"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreNotifications": {
+            "$ref": "./examples/GetPrivateStoreNotifications.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/offers/{offerId}/notifications": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/OfferIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_UpdateNotification",
+        "description": "Update notification for offer",
+        "parameters": [
+          {
+            "in": "body",
```

Comment:
The PUT payload definition needs to be the same as the PUT and GET response definition so someone can PUT what they GET.  This definition is just the property bag, not the full resource.  This applies to all PUTs where you are taking this properties definition instead of the resource definition

Reply:
Changed to new resources